{"name": "test_unsuccessful_login[chrome- or 1=1 -- --1212]", "status": "passed", "attachments": [{"name": "log", "source": "c9c2c878-ad4e-461e-b9e5-ea1e76931737-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_browser", "value": "'chrome'"}, {"name": "username", "value": "' or 1=1 -- -'"}, {"name": "password", "value": "1212"}], "start": 1622284250243, "stop": 1622284250765, "uuid": "3d05ae58-6fba-480d-bb55-18db835e5485", "historyId": "2c2626598a52618999f833b036198faa", "testCaseId": "1ce5752f9bf5ac0dd16171f6da2bde9d", "fullName": "test_cases.test_user_sign_in.Test_UserSignIn#test_unsuccessful_login", "labels": [{"name": "severity", "value": "normal"}, {"name": "story", "value": "Cloud Login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('add_logs_on_failure', 'get_browser')"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_user_sign_in"}, {"name": "subSuite", "value": "Test_UserSignIn"}, {"name": "host", "value": "DESKTOP-57RCRK3"}, {"name": "thread", "value": "10072-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_user_sign_in"}]}